src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzResultsContentPanel.java

```diff
@@ -116,7 +127,39 @@ public void itemStateChanged(ItemEvent e) {
 
         toolbar.add(Box.createHorizontalStrut(16));
         toolbar.add(showErrorsToggleButton);
-
+        
+        

```

Comment:
This lines should be changed (one of them removed and the trailing white spaces removed).